### PR TITLE
[TECH-433] Send failed build result to personal slack channel too

### PR DESCRIPTION
**Pull request by SamTheisens**
branch: fix/TECH-433-runner-should-also-report-errors-to-individual
 - add send to personal
 - fix slack markdown Screenshot 2023-05-25 at 09 49 51
 - speed up build by eliminating unnecessary github action
 - monorepo: https://github.com/Vandebron/Vandebron/pull/10517

### CODE REVIEW

File: src/mpyl/reporting/formatting/markdown.py [hunk 1]
```diff
@@ -83,7 +83,7 @@ def run_result_to_markdown(run_result: RunResult) -> str:
     elif run_result.failed_result:
         failed = run_result.failed_result
         result += f"For _{failed.project.name}_ at _{failed.stage}_ \n"
-        result += f" > {run_result.failed_result.output.message} \n\n"
+        result += f"\n```\n{exception}\n```\n"
```
Comment (jmvandebron) [inline src/mpyl/reporting/formatting/markdown.py:86]: We're in the `elif`, so exception is false here (test in line 80)? What content would it show?

**Review by jmvandebron — COMMENTED** [review]
I was already getting MPyL messages in Slack before these changes, the PR description should be 'show how to send build result to personal slack channel', as the PR only changes the `mpyl-example.py`.

One other comment.

File: src/mpyl/reporting/formatting/markdown.py (hunk 1 above)
Comment (SamTheisens) [reply src/mpyl/reporting/formatting/markdown.py:86]: Screenshot 2023-05-25 at 14 11 3👍
Indeed 👍 that's a bug

Comment (SamTheisens) [reply src/mpyl/reporting/formatting/markdown.py]: Fixed in ef76663ddfe24f54572224a0d8fe7a745e49eb3f

**Review by jmvandebron — APPROVED** [review]

**Pull request merged**